patches/last-opened.diff

```diff
@@ -10,7 +10,7 @@ Index: code-server/lib/vscode/src/vs/code/browser/workbench/workbench.ts
 ===================================================================
 --- code-server.orig/lib/vscode/src/vs/code/browser/workbench/workbench.ts
 +++ code-server/lib/vscode/src/vs/code/browser/workbench/workbench.ts
-@@ -350,19 +350,6 @@ class WorkspaceProvider implements IWork
+@@ -410,19 +410,6 @@ class WorkspaceProvider implements IWork
```

Comment:
Does this fix that weird space thing we were seeing?

Reply:
I am not sure if we are talking about the same thing but this fixes the warning output by `quilt push -a` since the patch is slightly off.

Reply:
yes, exactly!